wiki_service/linkHelper.py

```diff
@@ -0,0 +1,52 @@
+from typing import Dict, List, Tuple
+from .wikiEntry import MAX_LENGTH_OF_TITLE
+from pymongo import CursorType
+import re
+
+
+def createTitleToIdDict(entries: CursorType) -> Dict:
+    titleToID = {}
+    for article in entries:
```

Comment:
Maybe we can eliminate some error potential by trimming and lowercasing when we compare titles?